### ISSUE TITLE
Program is still only downloading a https file

### ISSUE THREAD

**Issue by JustPoko** [LordOfPolls/Rule34-Downloader#5]
![captura de tela 17](https://user-images.githubusercontent.com/31170721/52504310-3772b600-2bcf-11e9-972d-29f06fd7efed.png)
![captura de tela 18](https://user-images.githubusercontent.com/31170721/52504311-3772b600-2bcf-11e9-81fc-e6c28f92a367.png)

**Comment by LordOfPolls** [comment]
Can you talk me through exactly what youre doing. My testing doesnt show that
And have you downloaded the latest version from releases?

**Comment by JustPoko** [comment]
> Can you talk me through exactly what youre doing. My testing doesnt show that
> And have you downloaded the latest version from releases?

I downloaded the newest versions from releases, started it up and chose a folder, then chose a tag and confirmed the download. After a minute and a half, it said it was finished, but the only thing there is the https file. Maybe the problem is in my end, but I've tried it with different tags and different folders so I have no idea what it could be.

**Comment by LordOfPolls** [comment]
Can you attach the exe youre using and send it here?

**Comment by JustPoko** [comment]
Can't attach it, but here's a gdrive link for it
https://drive.google.com/drive/folders/1iQW034XVhWXYVcLcMXuTNhxMnD3wmWv3?usp=sharing

**Comment by LordOfPolls** [comment]
Just ran your exe worked on my end. Can you verify you can access rule34.xxx?

**Comment by JustPoko** [comment]
Can confirm rule34 is working

**Comment by LordOfPolls** [comment]
Interesting. Are you familiar with python?

**Comment by JustPoko** [comment]
I know what it is, but have no idea how to use it

**Comment by LordOfPolls** [comment]
Damn. Ill try to work out where the issue is occurring. Ill set up a VM and see if something on my system is making work for me and not you

**Comment by JustPoko** [comment]
Alright. Thanks for all the help!

**Comment by LordOfPolls** [comment]
I've been unable to replicate your issue. What tag(s) have you used?
![image](https://user-images.githubusercontent.com/22540825/52507262-e7a0e880-2be8-11e9-90dd-a13a079638a1.png)

**Comment by JustPoko** [comment]
Try using dako6995

**Comment by LordOfPolls** [comment]
![image](https://user-images.githubusercontent.com/22540825/52507876-b88b7680-2bea-11e9-8555-ba445e0a65de.png)

**Comment by LordOfPolls** [comment]
Does it show any errors or anything when running?

**Comment by JustPoko** [comment]
not at all

**Comment by LordOfPolls** [comment]
Hang on. Im going to write a new exe that doesnt clear the screen

**Comment by JustPoko** [comment]
alright

**Comment by LordOfPolls** [comment]
Try this and run it in command prompt
[rule34Downloader.zip](https://github.com/LordOfPolls/Rule34-Downloader/files/2847003/rule34Downloader.zip)

**Comment by JustPoko** [comment]
Alright I'll try it out

**Comment by JustPoko** [comment]
Nope, still the https file

**Comment by LordOfPolls** [comment]
Can i see what the code outputted in command line?

**Comment by JustPoko** [comment]
Hmm I made have found something, does the program usually auto quit a bit after it finishes?

**Comment by LordOfPolls** [comment]
Yeah. it waits 20 seconds then closes

**Comment by JustPoko** [comment]
oh nevermind then. Also do you mean this?
![captura de tela 87](https://user-images.githubusercontent.com/31170721/52508695-9e4a9c00-2bdc-11e9-9bfd-9879cd648bdd.png)

**Comment by LordOfPolls** [comment]
Yeah. Thats definitely not normal behavior. It looks like its getting stuck on a single file and looping. interesting. Gimmi a sec. gunna send you yet another exe if you dont mind XD

**Comment by JustPoko** [comment]
Don't mind at all

**Comment by LordOfPolls** [comment]
[rule34Downloader.zip](https://github.com/LordOfPolls/Rule34-Downloader/files/2847064/rule34Downloader.zip)

**Comment by JustPoko** [comment]
the result looks exactly like the last one, still the https

**Comment by LordOfPolls** [comment]
Im honestly stumped then. Not really sure the issue is

**Comment by JustPoko** [comment]
I'll try running it a couple of times with different tags, I'll let you know if I find anything out.

**Comment by JustPoko** [comment]
Thanks for all the help so far

**Comment by cremia4** [comment]
It seems that the problem still Persist. its downloading only a http file, what can we do to fix the problem?

**Comment by LordOfPolls** [comment]
As above, im unsure whats causing the issue for some users. All i know is that on every system ive tested on it works fine, and if i cant replicate the bug, i cant patch it

**Comment by cremia4** [comment]
Witch Are the Systems requirements or dependency for the program to run correctly on the system?

**Comment by petrolivro** [comment]
I have this same issue

**Comment by SaySayTakamura** [comment]
Havin the same error around here, i tried to run as admin and nothing yet, i´m going to try with different tags and see the results.

![1](https://user-images.githubusercontent.com/38867316/53802578-e41b2b80-3f20-11e9-93cc-2f99a9ba755a.PNG)

**Comment by SaySayTakamura** [comment]
I was serching in the program code and opened this link which in the code symbols the url for search of therms
![1](https://user-images.githubusercontent.com/38867316/53804360-15e2c100-3f26-11e9-8350-568de8b6b3d3.PNG)
